skymp5-server/cpp/server_guest_lib/PapyrusKeyword.cpp

```diff
@@ -0,0 +1,39 @@
+#include "PapyrusKeyword.h"
+
+VarValue PapyrusKeyword::GetKeyword(VarValue self,
+                                    const std::vector<VarValue>& arguments)
+{
+  if (arguments.empty()) {
+    spdlog::error("Keyword.GetKeyword - at least one argument expected");
+    return VarValue();
+  }
+
+  std::string keywordName = arguments[0].GetType() == VarValue::kType_String
+    ? (const char*)arguments[0]
+    : "";
+  std::transform(keywordName.begin(), keywordName.end(), keywordName.begin(),
+                 [](unsigned char c) { return std::tolower(c); });
+
+  for (size_t i = 0; i < keywords.size(); ++i) {
```

Comment:
I'm not sure it's 100% needed, but

could you please reverse loop order here?
```c++
for (auto it =  keywords.rbegin(); it != keywords.rend(); ++it)
```

https://www.tutorialspoint.com/cpp_standard_library/cpp_vector_rbegin.htm#:~:text=The%20C%2B%2B%20function%20std%3A%3A,moves%20towards%20beginning%20of%20vector.

Theoretically, it should prefer Keywords from files with greater indexes in case of the same name:
SweetPie.esp would be in priority over Skyrim.esm, etc

Also should be much faster in our use cases.